spiffworkflow-frontend/src/components/ProcessModelFileUploadModal.tsx

```
@@ -1,37 +1,31 @@
 
 interface ProcessModelFileUploadModalProps {
   showFileUploadModal: boolean;
   processModel: any;
-  onFileUpload: (event: any) => void;
   handleFileUploadCancel: () => void;
-  checkDuplicateFile: (event: any) => void;
-}
-interface FileUploadState {
-  filesToUpload: File[] | null;
+  checkDuplicateFile: (files: File[], forceOverwrite?: boolean) => void;
+  doFileUpload: Function;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)